custom/sponsors.md

```diff
@@ -4,7 +4,56 @@
 
 ## <span id="section:sponsors">❑ Sponsors</span>
 
-Development of Sanctuary is encouraged by the following generous supporters:
+Development of Sanctuary is funded by the following community-minded
+__partners__:
+
+  - <p>
+      <a id="logo-fink" href="https://www.fink.no/">
```

Comment:
@voxbono, is this your preferred `href`?

Reply:
yes, this is fine! :)